---
bibliography: manuscript.bib
csl: plos.csl
output:
  html_document: default
  word_document: default
  pdf_document: default
  github_document: default
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	message = FALSE,
	warning = FALSE,
	echo=FALSE,
	dpi=300,
	cache=FALSE
	)
```


```{r load_data}

	# Load data and analysis results generated by
	# manuscript_kernel.R
	load("manuscript.RData")
```


```{r preliminaries}
library( tidyverse )
library( magrittr )
library( viridis )
library( knitr )
library( gridExtra )

source( "functions.R" )

# A colorblind friendly palette from http://www.cookbook-r.com/Graphs/Colors_(ggplot2)/#a-colorblind-friendly-palette:
cbPalette <- c("#999999", "#E69F00", "#56B4E9", "#009E73", "#F0E442", "#0072B2", "#D55E00", "#CC79A7")

result_colors = c( cbPalette[2], cbPalette[4], cbPalette[1] )
names( result_colors ) = c( "Ctenophora-sister", "Porifera-sister", "Unresolved" )


```

# Rooting the animal tree of life

Yuanning Li^1,3^, Xingxing Shen^3^, Benjamin Evans^2^, Antonis Rokas^3^, and Casey W. Dunn^1^*

^1^Department of Ecology and Evolutionary Biology, Yale University

^2^Yale Center for Research Computing, Yale University

^3^Department of Biological Sceince, Vanderbilt University
\* Corresponding author, casey.dunn@yale.edu


## Abstract
Phylogenomic studies based on hundreds to thousand of genes have greatly improved our understanding of evolutionary relationship among major animal lineages. However, a considerable debate still remains regarding the first diverging animal lineage, with Ctenophora-sister and Porifera-sister emerging as the two primary hypotheses, comprising one of the most difficult node to resolve in animal tree of life. Here, we systematically explore the mechanisms that potentially cause the incongruent results from different studies by synthesizing data and results from all previous phylogenomic analyses, and perform new analyses in these data in a consistent manner to characterize the differences between studies. We find the gene sampling varies across studies, indicating that different regions of the genomes are sampled with little overlap among data matrices. Moreover, We find  Porifera-sister can only be recovered in several data matrices using site-heterogeneous CAT model with inclusive of more closely related outgroups, whereas Ctenophora-sister is recovered in other cases even with site-heterogeneous C60 models. We next quantify the support of phylogenetic signal over the two hypotheses and characterize substantial differences of phylogenetic distribution between site-homogenious and site-heterogeneous models. Rather than embracing one hypothesis over another, we hope our efforts provides an integrative overview of the challenge and provides direction for future studies to address difficult phylogenetic problems.

## Introduction

Phylogenomic analyses with genome-scale data have the potential to revolutionize our understanding of the tree of life. However, over the past decade, there has been considerable debate about the position of the root of the animal phylogeny, with Ctenophora-sister (comb jelly) and Porifera-sister (sponges) (Fig XXOverview) emerging as the two primary hypotheses. Historically, there was little debate about the root of the animal tree of life and Porifera-sister was widely accepted though rarely tested. In contrast to the lack of debate about the position of Porifera, there has long been uncertainty about the relationship of Ctenophora to other animals [@Wallberg:2004ws]. An understanding of the first diverging animal lineage is critical for scientists to reconstruct events that occurred  early in animal evolution, especially for complex structures such as cell types, nervous systems, and developmental patterns. However, a wide consensus of the position of animal has not been reached despite multiple efforts.

The first phylogenomic study to include ctenophores [@Dunn:2008ky] suggested a new hypothesis, now referred to as Ctenophora-sister, that ctenophores are our most distant animal relative. Since then many more studies have been published, some supporting Ctenophora-sister, some Porifera-sister, and some neither. As it has become clear that this is a very difficult phylogenetic challenge, and the problem has become better characterized, it has become an interesting test-case to phylogenetic biologists beyond those concerned with this particular biological problem. Work has been hindered, though, because it has been difficult to directly compare results across studies and synthesize findings to understand the broader patterns of support. Here we synthesize data and results from all previous phylogenomic analyses that tested Ctenophora-sister and Porifera-sister, and reanalyze these data using standardized methods, and perform new analyses to characterize differences between studies. We hope that this effort provides an integrative overview of the challenge and provides direction for future studies. We also hope that the work we have conducted here, including consolidating all the datasets in one place with consistent formats and species names, will enhance the technical value of this interesting question to methods-focused investigators that look to develop methods to address difficult phylogenetic problems.


![](figures/Figure_overview.png)
**Fig XXOverview.** (A) The Ctenophora-sister hypothesis posits that there is a clade (designated by the orange node) that includes all animals except Ctenophora, and that Ctenophora is sister to this clade. (B) The Porifera-sister hypothesis posits that there is a clade (designated by the green node) that includes all animals except Porifera, and that Porifera is sister to this clade. Testing these hypotheses requires evaluating the support for each of these alternative nodes. (C) The animals and their outgroup choice, showing the three progressively more inclusive clades Choanimalia, Holozoa, and Opisthokonta (designated by the blue node). (D) A ctenophore. (E) A sponge.


## Variation across studies

### Models of molecular evolution

Models of molecular evolution have several components that each consider different aspects of the evolutionary process. The models that have been used to model protein evolution in studies of the animal root have largely differed according to three components: the exchangeability matrix $E$, the rate of evolution, and the state equilibrium frequencies $\Pi$.

The exchangeability matrix $E$ describes the rate at which one amino acid changes to another. Exchangeability matrices have been used in the studies under consideration here include:

- F81 [@Felsenstein:1981vk] corresponds to equal rates between all states. The F81 matrix is also sometimes referred to as the Poisson matrix. It has no free parameters to estimate since all off-diagonal elements are set to 1.

- WAG [@Whelan:2001ds] is an empirically derived exchangeability matrix based on a dataset of 182 globular protein families. It has no free parameters to estimate since all off-diagonal elements are set according to values estimated from this particular sample dataset.

- LG [@Le:2008fp], like WAG, is an empirically derived exchangeability matrix. It is based on a much larger set of genes, and variation in rates across sites was taken into consideration when it was calculated. It has no free parameters to estimate since all off-diagonal elements are set according to values estimated from this particular sample dataset.

- GTR, the General Time Reversible exchangeability matrix, has free parameters for all off-diagonal elements that describe the exchangeability of different amino acids. It is constrained so that changes are reversible, *i.e.* the rates above the diagonal are the same as those below the diagonal. This leaves 190 parameters that must be estimated from the data long with the other model parameters and the phylogenetic tree topology. This estimation requires a considerable amount of data and computational power, but if successful has the advantage of being based on the dataset at hand rather than a different dataset (as for LG and WAG).

While the exchangeability matrix describes the relative rate of different changes between amino acids, the actual rate can be further scaled. There are a couple approaches that have been used in the studies considered here:

- Site homogeneous rates. The rates of evolution are assumed to be the same at all sites in the amino acid alignment.

- Gamma rate heterogeneity. Each site is assigned to a different rate class with its own rate value. This accommodates different rates of evolution across different sites. Gamma is used so commonly that sometimes it isn't even specified, making it difficult at times to know if a study uses Gamma or not.

The vector of equilibrium frequencies $\Pi$ describes the stationary frequency of amino acids. There are a few approaches that have been used across the studies considered here:

- Empirical site-homogeneous. The frequency of each amino acid is observed from the matrix under consideration and applied to homogeneously to all sites in the matrix.

- Estimated site-homogeneous. The frequency of each amino acid is inferred along with other model parameters, under the assumption that it is the same at all sites.

- CAT site heterogeneous [@Lartillot:2004dq]. Each site is assigned to a class with its own equilibrium frequencies. The number of classes, assignment of sites to classes, and equilibrium frequencies within the data are all estimated in a Bayesian framework.

- C10 to C60 [@si2008empirical]. 10 to 60-profile mixture models as variants of the CAT model under the maximum-likelihood framework.

- Six-state amino acid recoding. Amino acids are recoded into six groups based on function to account for both compositional heterogeneity and substitution saturation. Several recoding strategies have been proposed, including Dayhoff 6-state recoding, S&R 6-state recoding, KGB 6-state recoding based on exchangeability matrix of models.

Models can be assembled by selecting different options for all these different components. The models that are applied in practice area heavily influenced by engineering and computational costs, as well as convention. For example, on the questions considered here, F81 and GTR exchangeability matrices have only been used in combination with CAT site heterogeneous models of equilibrium frequency. LG and WAG exchangeability matrices have only been used with site homogeneous estimates of equilibrium frequency. This is further confused by the abbreviations that are used for models. Papers often discuss CAT and WAG models as if they are exclusive, but these particular terms apply to non-exclusive model components-- CAT refers to variation across sites and WAG a particular exchangeability matrix. CAT is generally shorthand for F81+CAT and WAG is shorthand for WAG+homogeneous equilibrium frequency estimation. One could, though, run a WAG+CAT model.

To avoid confusion on this point, we always specify the exchangeability matrix first, followed by modifiers that describe the accommodation of heterogeneity in equilibrium frequencies (*e.g.*, CAT) or rate (*e.g.*, Gamma). If there are no modifiers, then it is implied that site homogeneous models are used.

### Gene sampling

High-throughput sequencing is leading to exciting new data matrices for phylogenomic analyses, potentially including hundreds or thousands of genes (*e.g.*, [@whelan2017ctenophore; @simion2017large]). Although the number of sampled genes might be different across studies, it is reasonable to expect that the sampled genes should be conserved across animal genomes and most data matrices should share a core set of orthology genes (OGs) (*e.g.*, BUSCO genes). Surprisingly, gene composition and matrice overlap across different studies has been rarely assessed, since most studies relied on their own bioinformatic pipeline for determination of OGs.  

### Outgroup and ingroup taxon sampling

Outgroup selection (the taxa used to root the ingroup taxa) can potentially affect the phylogenetic inference of the root position of animal phylogeny (**Fig XXOverview.C**). Several studies have progressively removed more distantly related outgroup taxa to test the effect of outgroup selection to ingroup topology [@ryan2013genome; @pisani2015genomic]. Three progressively more inclusive clades have often been investigated, including Choanomalia (Metazoa plus most closely related Choanoflagellata), Holoza (Choanimalia plus more distantly related holozoans) and Opisthokonta (Holozoa + Fungi). It has been suggested the inclusion of more distant related (e.g. Opisthokonta) outgroup taxa tend to recover Ctenophora-sister, whereas high support of Porifera-sister often recovered only when site-heterogeneous CAT models were used. In contrast of outgroup selection, sensitivity to ingroup sampling has received less attention than sensitivity to outgroup sampling. This may be because results have tended to be more sensitive to outgroup sampling.


### Data-recoding

Compositional heterogeneity (lineage-specific differences in amino acid frequencies) was long thought has a major effect in resolving the animal tree of life, especially for the animal root position. Feuda et al. 2017 [@feuda2017improved] has proposed to recode the full set of twenty amino acids into six groups that have more frequent evolutionary changes within groups than between groups based on different exchangeability matrices. By discarding information on which specific amino acid is found at each site in each species and instead focusing on which groups those amino acids belong to, they sought to reduce the impact of differences between species in which particular amino acids within those groups are most frequent. By reanalyzing Chang et al. 2015 [@chang2015genomic] and Whelan et al. 2015 [@whelan2015error], they found that Porifera-sister was strongly favored under all recoding strategies using GTR-CAT models [@feuda2017improved]. However, the performance of the recoding methods has never been tested empirically against the non-recoding method, and a recent simulation study has suggested that non-recoding approaches significantly outperformed 6-state recoding strategies [@hernandez2019six]. Thus, it is important to evaluate the performance of data recoding method verses non-recoding approaches in past empritical matrices .

## Overview of published analyses
Here, we synthesize data and results from all previous phylogenomic analyses that tested Ctenophora-sister and Porifera-sister, including 17 data matrices from 12 studies (Table 1).

Table 1.
```{r overview data matrices}
	kable(datasets)

```


### Matrix taxon composition

Taxa sampling plays a critical role in phylogenetic inference, especially for the phylogenetic position that needs to be explored. However, several data matrices have relatively skewed taxon sampling, with more than 50% taxa sampled in bilaterians compared to other major animal lineages (Fig XXTaxon_composition). These skewed data matrices include two studies constructed from ESTs (Dunn2008 [@Dunn:2008ky] and Hejnol2009 [@hejnol2009assessing]) and one study from whole-genome data (Borrowiec2015 [@borowiec2015extracting] ). Importantly, some earlier studies only include less than five ctenophore species. The rest data matrices have relatively balanced taxon composition, although the total number of sampled taxon varies a lot. In general, increasing taxon sampling is broadly accepted to improve phylogenetic inference and aid in the placement of animal root position, and it should be noted that Whelan2017 [@whelan2017ctenophore] and Simion2017 [@simion2017large] have much broader taxon sampling than other data matrices: Simion2017 has the most taxon sampling, and Whelan2017 contains greater ctenophore taxon sampling.


```{r taxon_rectangles }

no_subsets = unlist(lapply(sequence_matrices, function(x) !grepl("only|_D", x@matrix_name)))

taxon_rectangles = lapply(
	sequence_matrices[no_subsets],
	function ( sequence_matrix ){
		clade_rects(sequence_matrix)
	}
) %>%
bind_rows()


clade_colors = viridis( length(clades) )
names( clade_colors ) = clades

ggplot(data=taxon_rectangles) +
	scale_x_continuous(name="x") +
	scale_y_continuous(name="y") +
	geom_rect( mapping=aes(xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax, fill=clade )) +
	# scale_fill_manual(guide = guide_legend(reverse=TRUE))
	# scale_fill_manual(values = clade_colors, guide = guide_legend(reverse=TRUE))
	scale_fill_discrete(guide = guide_legend(reverse=TRUE)) +
	facet_wrap( ~ matrix )

```



**Fig XXTaxon_composition.** Each of the primary matrices considered here, color coded by taxon sampling. Horizontal size is proportional to the number of genes sampled, vertical size to the number of taxa sampled.

### Matrix gene composition and overlap

Gene sampling plays an essential role in phylogenomic analyses, and most often rely on the concatenation of sampled genes to tackle the incongruence of the topology among individual gene trees. The basic idea is that the concatenation assumes that phylogenetic signal from genes that do not share the species phylogeny will be overwhelmed by the signal from the majority of genes whose genealogy mirrors that of the species evolutionary history [@lanier2012recombination]. However, large phylogenomic data matrices potentially contain genes with conflicting or weak phylogenetic signal that can lead to incongruence phylogenomic results, especially when analyzing deeper node represents ancient divergence in the tree [@salichos2013inferring]. Thus, it is important to evaluate the gene composition and matrix overlap to explore congruence of gene sampling across different matrices. It should be noted that early studies relied on EST via Sanger sequencing contains large missing data compared to other ones.

We first used the Metazoa BUSCO dataset to evaluate gene composition from all data matrices and the result is unexpected. The animal BUSCO dataset contains 978 single-copy OGs (identified from 64 animal taxa) that are most likely shared by most animal genomes, and are commonly used to assess the completeness of genome or transcriptomic assemblies. Morever, the BUSCO dataset also are frequently used as phylogenomic markers in a varietly of studies. Thus, it is reasonable to expect that these conserved genes should present at a higher percentage than non-BUSCO genes in most matrices given their nature of conserveness and high-expression properties. However, to our surprise, we found that gene partitions from all matrices comprised less than 50% of BUSCO genes, even with several data matrices contain less than 20 percent (Fig. XXBUSCO_annotations).

We furthered BLAST gene partitions to SwissProt database

Previous reanalyses have mainly focused on ourgroup selection and/or model fits in different data matrices, and the matrix overlap across different studies has been rarely investigated. Our initial intent of comparing matrix overlap across matrices was to construct a new matrice with shared taxon and gene sampling to test specific hypotheses about differences in support. The hypothesis is that although the bioinformatics pipeline used for OG determination is different, most larger data matrices should share a core set of OGs. However, our results show that surprisingly little overlap across all the data matrices from different studies (Fig XXAlignment overlap). These results indicated that different studies essentially sampled different regions of the genomes with little overlap. One thing should be noted that




```{r gene_composition}

busco_overrepresented


p1 = ggplot( matrix_summary ) +
	geom_point( aes(x=n_partitions, y=n_busco_partitions, col=manuscript) ) +
	geom_abline(slope=1,intercept=0) +
	coord_fixed()

matrix_summary$matrix = factor( matrix_summary$matrix, levels=c("Dunn2008", "Philippe2009", "Hejnol2009", "Nosenko2013_nonribo_9187","Nosenko2013_ribo_11057","Nosenko2013_ribo_14615","Ryan2013_est", "Moroz2014_3d", "Chang2015", "Borowiec2015_Best108" , "Borowiec2015_Total1080", "Whelan2015_D1", "Whelan2015_D10", "Whelan2015_D20", "NA", "Whelan2017_full", "Whelan2017_strict") )

p2 = ggplot( matrix_summary ) + geom_point( aes(x=matrix, y=n_busco_partitions/n_partitions*100, col=manuscript) ) + 	theme_classic() + theme(axis.text.x = element_text(angle=45, hjust=1))


```



**Fig XXBUSCO_annotations.** The number of partitions with BUSCO annotations in each matrix, relative to the number of partitions.


**Fig XXGene_composition.** Each of the primary matrices considered here, color coded by the types of genes sampled (XX Ribosomal proteins, etc). Horizontal size is proportional to the number of genes sampled, vertical size to the number of taxa sampled.


```{r matrix_overlap, fig.height = 12, fig.width = 20}

matrix_overlap_rectangles =
	lapply(sequence_matrices[no_subsets], function(x) lapply(sequence_matrices[no_subsets], function(y) overlap_rects(x,y))) %>%
	unlist(recursive=FALSE) %>%
	bind_rows()

ggplot(data=matrix_overlap_rectangles) +
	scale_x_continuous(name="Genes") +
	scale_y_continuous(name="Species") +
	geom_rect( mapping=aes(xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax, fill=MSA ), alpha=0.5) +
	facet_grid( matrix_1 ~ matrix_2, scales="free" ) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1), strip.text.y = element_text(angle = 0), strip.text.x = element_text(angle = 60))

```


**Fig XXAlignment overlap.** Pairwise overlap between each of the primary matrices considered here. Horizontal size is proportional to the number of genes sampled, vertical size to the number of taxa sampled. The horizontal intersection shows the proportions of shared genes, the vertical intersection shows the proportions of shared taxa.





### Support for Porifera-sister and Ctenophora-sister

Here we summarized all the 167 previous phylogenetic analyses from all studies (Fig. support_published_analyses and Table perviouly_published_analyses). We found that all past phylogenomic analyses strongly support of Ctenophora-sister once site-homogeneous models were used, no matter what outgroup choice, phylogenetic inference, recoding methods are used. We found that support for Porifera-sister increases with the exclusion of more distantly related outgroups with the use of site-heterogeneious CAT models. It should be noted that the convergence were not reached or posterior probability is not well supported in many BI analyses. Moreover, the strong support of Porifera-sister are mainly from analyses using Poisson+CAT and recoding methods with site-heterogeneous models. Moreover, only a few past analyses with the most complex GTR+CAT model are converged. However, for the ones that are converged with GTR+CAT model, Porifera-sister is not always supported since several analyses strongly in favor Ctenophora-sister when choanozoa are used as outgroups.

The use of site-heterogeneous CAT is an infinite mixture model has long been suggested as the best approach to account for systematic errors, particular for data saturation, especially in the case of long-branch attraction (LBA) (Lartillot and Philippe, 2004). The GTR-CAT model often received the best-model fit from cross validation or posterior predictive analyses compared to site-homogeneous or Poisson-CAT models. However, the GTR+CAT model is extremely computational demanding and not feasible in large matrices so many studies only include Poisson+CAT models in their analyses. In contrast, a recent study has suggested that Poisson+CAT model did not outperform site-homogeneous with data partitioning or GTR + CAT models and in some cases can infer incorrect topology based on the simulation analyses. The author also raised the issue of the potential overparameterization in CAT models in some cases. Nevertheless，the debate over the site-heterogeneous vs. site-homogeneous models and/or inclusion or exclusion more distantly related outgroups, constitutes the probably one of the most controversial nodes in the animal tree of life.


```{r support_published_analyses}

analyses_published %>%
	ggplot( aes(x=model_combined, y=clade, col=result, shape=inference) ) +
		geom_jitter( width = 0.20, height = 0.20, alpha=0.7, size=2 ) +
		scale_colour_manual(values = result_colors) +
		theme_classic() + theme(axis.text.x = element_text(angle=45, hjust=1))+
   xlab("model complexity")
```


**Fig support_published_analyses.**  A total of `r nrow(analyses_published)` analyses were transcribed from the literature (Table perviouly_published_analyses.tsv).



## New analyses of published matrices


```{r support_new_analyses}

analyses_new %>%
	ggplot( aes(x=model_summary, y=clade, col=result, shape=inference) ) +
		geom_jitter( width = 0.20, height = 0.20, alpha=0.7, size=2 ) +
		scale_colour_manual(values = result_colors) +
		theme_classic() + theme(axis.text.x = element_text(angle=45, hjust=1))
```



One of the challenges of interpreting support for the placement of the animal root across studies is that different programs, software versions, and settings have been used across studies, and phylogenetic analysis decisions have been approached in very different ways. Here we reanalyze the primary matrices from each study under consistent conditions with IQtree under a panel of evolutionary models. We selected this tool because it has greater model flexibility, computational time and accuracy than other tools in maximum likelihood (ML) framework. Importantly, we also include a CAT-like, site-heterogeneous C10 - C60 (C) models that are implemented in both ML framework in our analyses and it is interesting to compare results between C model to CAT models since it has never been used in any matrices analyzed here.

We first tested a variety of models for each matrix and compared the relative fit of site-homogeneous and site-heterogeneous C models using ModelFinder in IQtree. We found in all cases, C60+ WAG/LG models fit these matrices better than the site-homogeneous models under BIC criteria and we then inferred support under the best-fit model from IQtree. It should be noted that we didn't include GTR + C models in model testing since it is computational too expensive. The GTR+C models requires to estimate up to over 10,000 (e.g. GTR+C60 contains 234*60 parameters) free parameters inferred from matrices, whereas only several hundred parameters needs to estimate in site-homogeneous + C models. We then analyzed every matrix under a panel of standard site-heterogeneous and site-homogeneous models, including WAG+G, GTR+G and Poisson+C60+F+G. Moreover, we also used the best-fit model identified above with the removal of all distantly related outgroups and only include Choanoflagellata as outgroups.

With the exception of Moroz2014 matrix that supports neither Porifera or Ctenophora-sister, every analysis conducted herein choice strongly supported the ctenophore-sister hypothesis, even with site-heterogeneous C models with the most closely related outgroups.



```{r modelfinder}

analyses_new %>%
	filter(modelfinder==TRUE) %>%
	select( matrix, clade, result, model_summary ) %>%
	kable()

```

Table XXModelfinder. The models selected by modelfinder for each matrix.


### Comparison of iqtree and phylobayes results

Site heterogeneity in equilibrium frequency has been a major concern in tests of Ctenophora-sister and Porifera-sister. This has been addressed with CAT models. iqtree provides a new family of C models that also address site heterogeneity. Given the extensive computational cost and concerns about overparameterization of CAT models, we compared iqtree C results to CAT results for a subset of matrices to see if they give consistent results. This would be of technical interest because it would reduce the cost of accommodating compositional heterogeneity in future analyses. For computational efficiency, we first only applied Poisson+CAT models in every matrix with only Choanoflagellata as outgroup. It should be noted that most Phylobayes runs were converged, several large matrices have not reached convergence after at least a month's computational time.

Interestingly, we found strong supports for both hypotheses when using Poisson-CAT model in Choanimalia matrices. Consistent with previous study, we recovered the strong support of Porifera-sister in matrices of Philippe2008, Ryan2013, and Whelan2015. Moreover, we also found a strong support of Porifera-sister in two Whelan2017 matrices and a weaker support of subsampled matrix of Simion2017. Similar to ML analyses, we found no strong support for either hypothesis in Moroz2013_3d matrix. For the matrices have conflicted results between Poisson-CAT and C models, we further ran Poisson+CAT models with inclusion of more distantly related outgroups to evaluate how outgroup choice will affect phylogenetic inference under CAT model. Consistently with past study, we found strong support of Ctenophora-sister in all anaylzed Opisthokonta matrices, indicating that Ctenophora-sister is also favored in all matrices with more distantly related taxa are used as outgroups.

### Phylogenetic signal

To further explore the effect of site-homogeneous or site-heterogeneous models and outgroup choices in on the animal-root position, we next quantify the support of phylogenetic signal over two alternative hypotheses (T1: Ctenophora-sister (Fig. 1A); T2: Porifera-sister (Fig. 1B)) to three key data matrices with two outgroup choice (Choanimalia and Opisthokonta), including Philippe2009, Ryan2013_est and Whelan2017_full (Fig XXPhylogenetic signal). By calculating gene-wise log-likelihood scores between T1 and T2 for every gene (\Delta$GLS) or site (\Delta$SLS)in each matrix, we found that Ctenophora-sister had the higher proportions of supporting genes or sites in every analysis. These results are consistent with best-likelihood tree always favors Ctenophora-sister over Porifera-sister in all IQtree analysis.

Surprisingly, we found significant differences of phylogenetic distribution between site-homogenious and site-heterogeneous models. We found that Ctenophora-sister had the higher proportions of supporting genes when site-homogeneous models were used (52.34% to 66.51%), and the outgroup choice has little impact on the distribution of the support phylogenetic signal. This finding is consistent with the past study that majority of phylogenetic signal is strongly favored in Ctenophora-sister hypothesis with site-homogeneous models [@shen2017contentious].  

In contrast to site-homogenious models, the distribution of phylogenetic signal over the animal-root position has never been quantified with site-heterogeneous models. Surprisingly, except for Whelan2017_Holozoa matrix, we found that the phylogenetic signal decreases in many genes using C models compared to WAG models, especially in two matrices from Ryan_2013 that over 30% of genes changed from strong to week signal (\Delta$lnL<2). It is not clear the reason why the phylogenetic signal decreases in many genes with a better fitting model as suggested by model testing (control test will be conducted here, by changing monophyletic of bilaterian vs. non-monophyletic of bilaterians...).



```{r phylogenetic_signal}
au_tests %>%
  unite(Matrix, c(manuscript, matrix)) %>%
  group_by(Matrix, result) %>%
  count() %>%
  group_by(Matrix) %>%
  mutate(percent = prop.table(n)) %>%

ggplot(aes(x = Matrix, y = percent, fill=result)) + 
  geom_bar(stat = "identity") +
  #scale_y_continuous(limits = c(0,1), expand = c(0, 0))+
  theme(axis.text.x = element_text(angle=45, hjust=1)) +
  scale_fill_manual(values=c("#E69F00", "#ebd196","#a2e8d5","#009E73"))

```
**Fig XXPhylogenetic signal.** The distribution of phylogenetic signal for two alternative topological hypotheses on the earliest-branching animal lineage. Proportion of genes supporting each of two alternative hypotheses for each of

## The current state of understanding

### Interpretting variation support

External criteria, eg posterior predictive scores, model fit etc

## Next steps

## Conclusion

## Methods

All files and bioinformatic commmands associated with this analysis are available at https://github.com/caseywdunn/animal_root

### Data selection and wrangling

We retreived matrices from each publication (Table XX), storing the raw data in this manuscript's version control repository. We manually edited some minor formatting to make the batch processing of the matrices *en masse*, e.g. standardizing the formatting of charset blocks. All changes made are tracked with git.

### Matrix comparison and annotation

#### Taxon name reconciliation

We programatically queried the NCBI Taxonomy database to standardize names of samples in each matrix. We also use a table where manual entries were needed (Supp Table XX, reconciliation/taxonomy_info/manual_taxonomy_map.tsv), e.g. authors of original matrix indicate species name in original manuscript. For a table summarizing all samples and their new or lengthened names, see Table XX(reconciliation/taxonomy_info/taxon_table.tsv).

#### Sequence Comparisons

Using the partition files for each matrix, we isolated each sequence for each taxon from each partition. Because many of the matrices had been processed by the original authors to remove columns that are poorly sampled or highly variable, these matrix-derived sequences can have deletions relative to the actual gene sequences.

We used DIAMOND [@Buchfink:2014] to compare each sequence to all others using default diamond blastp parameters. We further filtered DIAMOND results such that we retained hits for 90% of partitions (pident > 78.0, eValue < 1e-15, no self->self). We ran BUSCO with default parameters for all sequences against the provided metazoa gene set. We also ran a BLAST+ blastp search against the SwissProt [cite] database, filtering such that we retain at least one hit for ~97% of partitions (pident > 50.0, eValue < 1e-15).

#### Partition Network

We used the sequence similarity comparisons described above to compare partitions.

We constructed a network with Python and NetworkX [@Hagberg:2008] v2.2 where each node is a partition and each edge represents a DIAMOND sequence-to-sequence match between sequences in the partitions. We extracted each connected component from this network. We further split these components if the the most connected node (i.e. most edges) had two times more the standard deviation from the mean number of edges in the component it is a member of and if removing that node splits the component into two or more components. We then decorated every node in the partition network with the most often found SwissProt BLAST+ result and BUSCO results to see which components contain which classes and families of genes. See Table XX [partition_network_summary table] for a summary tally of each part of the comparison.


### Phylogenetic analyses

#### Phylogenetic analyses in IQtree
To investigate the phylogenetic hypotheses and distribution of phylogenetic signal in studies aiming to elucidate the root of animal position, we considered 17 data matrices from all phylogenomic studies that were constructed from EST, transcriptomic or genomic data (Table). Because different choices of substitution models could largely influence phylogenetic inference of the placement of the root of animal tree of life (e.g. site-heterogeneous vs. site-homogeneous models), we first investigated model-fit from each matrix using ModelFinder [@kalyaanamoorthy2017modelfinder] in IQtree [@nguyen2014iq], including site-heterogenous C10 to C60 profile mixture models as variants of the CAT models in ML framework (C10-C60 model were included for model comparison via -madd option). We included models that are commonly used in previous analyses, including site-homogeneous poisson, WAG, LG, GTR models plus C models in the model testing. For computational efficiency, the GTR with C models were not included in model testing since it requires to estimate over 10,000 parameters. For large matrices like Hejnol2009, Borrowiec2015 and Simion2017, model testing is also not computational feasible so only LG+C60 models were used since LG/WAG+C60 models were suggested as the best-fit model in all other matrices.

We then reanalyzed each matrix under a panel of evolutionary models, including WAG+G, GTR+G, poisson+C60 and associated best-fit model identified above. Nodal support was assessed with 1000 ultrafast bootstrap replicates for each analysis. Because of the large size of Hejnol2009 and Simion2017, it was not computationally feasible to analyze the whole matrix using the C model or CAT site-heterogeneous models. To circumvent his limitation, we reduced the data size from their full matrices to facilitate computational efficiency for site-heterogeneous models. For Hejnol2009 matrix, we instead used the 330-gene matrix constructed by Hejnol *et al*. 2009 [@hejnol2009assessing], since the main conclusion for their study is based on this subsampled matrix; For Simion2017 matrix, we only included most complete 25% of genes (genes that were present in less than 79 taxa were removed; 428 genes were kept). It should be noted that the main conclusion of Simion et al. 2008 was also based on selection of 25% of genes for their jackknife approach.

#### Outgroup taxa sampling with C10-C60 and CAT models
Because different choices of outgroups could also affect phylogenetic inference as suggested in previous analyses, we parsed the full data matrices into three different types of outgroups: Choanimalia , Holozoa and Opisthokonta. These datasets include the same set of genes but differ in the composition of outgroup species. Choanimalia only includes choanofagellates as outgroup; Holozoa also includes more distantly related holozoans; Opistokonta also includes Fungi. For each Choanimalia data matrice, both C models IQtree and Poisson-CAT models in PhyloBayes were conducted. The maximum likelihood analysis was performed using the best-fit substitution model identified as above and nodal support was assessed with 1000 ultrafast bootstrap replicates using IQtree. Moreover, bayesian inference with the site-heterogeneous poisson-CAT model was done with PhyloBayes MPI. To minimize computational burden, CAT-GTR models were only performed in the.

For results of choanozoa matrices indicated a strong support that sponges are the sister group to the remaining Metazoa using CAT model, bayesian inference with Poisson-CAT model was also conducted to Holozoa and Opisthokonta data matrices with the same settings as above. For all the analyses with Poisson+CAT models in PhyloBayes, two independent chains were sampled every generation. Tracer plots of MCMC runs were visually inspected in Tracer v1.6 to assess stationarity and appropriate burn-in. Chains were considered to have reached convergence when the maxdiff statistic among chains was below 0.3 (as measured by bpcomp) and effective sample size > 50 for each parameter (as measured by tracecomp). A 50% majority‐rule consensus tree was computed with bpcomp, and nodal support was estimated by posterior probability. For those matrices that were not converged, PhyloByes analyses were run for at least one month.

#### phylogenetic distribution of support
To investigate the distribution of phylogenetic signal in data matrices, we considered three major data matrices from 3 studies that had different topology between ML and BI using CAT model in our reanalysis, including Philippe2008, Ryan2013, and Whelan_2017 data matrices. We examined two hypotheses: Ctenophora-sister and Porifera-sister to rest of metazoans, under both ML and BI frameworks with different outgroup schemes (Choanomalia and Opisthokonta). For ML analysis in each dataset, site-wise likelihood scores ere inferred for both hypotheses using IQtree (option -g) with the same best-fit model identified above, and site-homogeneous WAG models. The two different phylogenetic hypotheses passed to IQtree (via -z) were the corresponding tree that the ctenophore as the sister lineage tree and the corresponding tree that was modified to have sponges as the sister to all other metazoans. The constraint trees were conducted in a customized R script. The numbers of genes and sites supporting each hypothesis were calculated with IQtree output and Perl scripts from Shen et al. 2017 [@shen2017contentious]. For BI analysis, we only considered the Philippe_2009 and Whelan_2017 datasets.


#### Performance of data recoding



## Ackowledgements

We thank the Yale Center for Research Computing for use of the research computing infrastructure, specificaly the Farnam cluster.



## Author contributions




\pagebreak

# Supplemental Information

## Details of published analyses

### Dunn *et al.* 2008

Dunn *et al.* [@Dunn:2008ky] added Expressed Sequence Tag (EST) data for 29 animals. It was the first phylogenomic analysis that included ctenophores, and therefore that could test the relationships of both Ctenophora and Porifera to the rest of animals. It was also the first phylogenetic analysis to recover Ctenophora as the sister group to all other animals.

The data matrix was constructed using a semi-automated approach. Genes were translated into proteins, promiscuous domains were masked, all gene sequences from all species were compared to each other with blastp, genes were clustered based on this similarity with TribeMCL [@Enright:2002uq], and these clusters were filtered to remove those with poor taxon sampling and high rates of lineage-specific duplications. Gene trees were then constructed, and in clades of sequences all from the same species all but one sequence were removed (these groups are often due to assembly errors). The remaining gene trees with more than one sequence for any taxon were then manually inspected. If strongly supported deep nodes indicative of paralogy were found, the entire gene was discarded. If the duplications for a a small number of taxa were unresolved, all genes from those taxa were excluded. Genes were then realigned and sites were filtered with Gblocks [@Castresana:2000vy], resulting in a 77 taxon matrix. Some taxa in this matrix were quite unstable, which obscured other strongly-supported relationships. Unstable taxa were identified with leaf stability indices [@Thorley:1999kg], as implemented in phyutility [@Smith:2008gb], and removed from the matrix. This resulted in the 64-taxon matrix that is the focus of most of their analyses. Phylogenetic analyses were conducted under the F81+CAT model in phylobayes [@Lartillot:2004dq], and under the WAG model in MrBayes [@Ronquist:2003hx] and RAxML [@Stamatakis:2006wc].

Regarding the recovery of Ctenophora-sister, the authors concluded:

> The placement of ctenophores (comb jellies) as the sister group to all other sampled metazoans is strongly supported in all our analyses. This result, which has not been postulated before, should be viewed as provisional until more data are considered from placozoans and additional sponges.

Note that there was, in fact, an exception to strong support. An analysis of the 40 ribosomal proteins in the matrix recovered Ctenophora-sister with only 69% support. This study did not include *Trichoplax*.

### Philippe *et al.* 2009

Philippe *et al.* 2009 [Philippe:2009hh] assembled an EST dataset for 55 species with 128 genes to explore phylogenetic relationship of early diverging animals. The phylogenetic analysis using Poisson-CAT model strongly supported Porifera-sister, and ctenophores was sister to cnidarians to form the "coelenterate" clade. The authors concluded:
> The resulting phylogeny yields two significant conclusions reviving old views that have been challenged in the molecular era: (1) that the sponges (Porifera) are monophyletic and not paraphyletic as repeatedly proposed, thus undermining the idea that ancestral metazoans had a sponge-like body plan; (2) that the most likely position for the ctenophores is together with the cnidarians in a ‘‘coelenterate’’ clade.

### Hejnol et al. 2009

Hejnol *et al.* 2009 added EST sequences from seven taxa, and a total of 94 taxa were included in the final data matrix. The orthology inference was largely simialr to Dunn et al. 2008. Two datsets were constructed, one with 844 genes and the other with 330 genes. Maximum likelihood analyses from both datasets strongly supported Ctenophora-sister.

### Pick *et al.* 2010

Pick *et al.* [@Pick:2010eb] sought to test whether Ctenophora-sister was an artefact of insufficient taxon sampling. They added new and additional published sequence data to the 64-taxon matrix of Dunn *et al.* [@Dunn:2008ky]. The new taxa included 12 sponges, 1 ctenophore, 5 cnidarians, and *Trichoplax*. They further modified the matrix by removing 2,150 sites that were poorly sampled or aligned. They considered two different sets of outgroups: Choanoflagellata (resulting in Choanimalia) and the same sampling as Dunn *et al.* (resulting in Opisthokonta).

All their analyses were conducted under the F81+CAT+Gamma model in phylobayes [@Lartillot:2004dq], in both a Bayesian framework and with bootstrapping. All analyses have the same ingroup sampling and site removal so it isn't possible to independently assess the impact of these factors. Analyses with Choanimalia sampling recovered Porifera-sister with 72% posterior probability (PP) and 91% bootstrap support (BS). With broader Opisthokonta sampling, support for Porifera-sister is 84% PP. This is an interesting case where increased outgroup sampling leads to increased support for Porifera-sister.

The authors argue that previous results supporting Ctenophora-sister "are artifacts stemming from insufficient taxon sampling and long-branch attraction (LBA)" and that "this hypothesis should be rejected". Although the posterior probabilities supporting Porifera-sister are not strong, they conclude:

> Results of our analyses indicate that sponges are the sister group to the remaining Metazoa, and Placozoa are sister group to the Bilateria

They also investigated saturation, and conclude that Dunn *et al.* [@Dunn:2008ky] is more saturated than Philippe *et al.* 2009 [Philippe:2009hh]. Note that the Pick *et al.* [@Pick:2010eb] dataset is not reanalyzed here because partition data are not available, and due to site filtering the partition file from Dunn *et al.* [@Dunn:2008ky] cannot be applied to this matrix.


### Ryan *et al.* 2013
Ryan *et al.* 2013 sequenced the first ctenophore genome of *Mnemiopsis leidyi*. With the genome resources of *M. leidyi*, the authors constructed two phylogenomic datasets: a "Genome set" based on 13 animal genomes and a "EST Set" that also included 59 animals. They analyzed both matrices by site-homogeneous GTR+Gamma model and Poisson-CAT model with three sets of outgroup sampling to evaluate the effect of outgroup selection to the ingroup topology. The results strongly supported Ctenophora-sister in all datasets they analysed using site-homogeneous model. The Poisson + CAT model of the genome dataset strongly supported of a clade of Ctenophora and Porifera as the sister group to all other Metazoa and Bayesian analysis on the EST dataset did not converge after 205 days.  

### Moroz *et al.* 2014

Moroz *et al.* 2014 sequenced the second ctenophore genome *Pleurobrachia bachei* to explore the phylogenetic relationship of Metazoa. All phylogenetic analyses strongly supported Ctenophora-sister with different taxon and gene sampling using WAG site-homogeneous model.

### Whelan et al. 2015
Wehlan *et al.* 2015 constructed a new phylogenomic dataset by eight new transcripomic data and investigated a range of possible sources of systematic error under multiple analyses (e.g. long-branch attraction, compositional bias, fast evolving genes, etc.). Putative orthologs were determined of each species using HaMStR using the model organism core ortholog set (~ 1000 orthologs) and subsequetly removal of genes with too much missing data and potential paralogs.  The authors further filtered the full dataset to 24 sub-datasets by filtering genes with high long-branch scores; genes with high RSFV values; genes that are potential paralogs; fast evolving genes and progressively removal of outgroups. All the maximum likelihood analyses with site-homogeneous model and PartitionFinder strongly suggested Ctenophora-sister. CAT-GTR models only used in least saturated dataset 6 and 16 also strongly supported Ctenophora. Regarding the recovery of Ctenophora-sister, the authors concluded:
> Importantly, biases resulting from elevated compositional heterogeneity or elevated substitution rates are ruled out. Placement of ctenophores as
sister to all other animals, and sponge monophyly, are strongly supported under multiple analyses, herein.

### Chang et al. 2015

Chang *et al.* 2015 was originally used to explore phylogenetic position of Myxozoa in Cnidaria but also sampled broadly across the breadth of animal diversity. The authors constructed a dataset with 200 protein markers based on Philippe et al. 2011 with 51,940 amino acids and 77 taxa. Both site-heterogeneous Poisson-CAT and site-homogeneous GTR models strongly supported Ctenophora-sister.

### Pisani *et al.* 2015

Pisani *et al.* 2015 reanalyzed representative datasets that supported Ctenophora-sister, including Ryan2013, Moroz2014 and Whelan2015 datasets. It was the first study showing that progressively removal of more distantly related outgroups could largely affect phylogenomic inference of animal-root position. The authors suggested that the inclusion of outgroups very distant from the ingroup can cause systematic errors due to long-branch attraction. They found strong support of porifera-sister when only include Choanozoa as outgroup in these datasets using site-heterogeous CAT model. Regarding the recovery of Porifera-sister, the authors concluded:
>Our results reinforce a traditional scenario for the evolution of complexity in animals, and indicate that inferences about the evolution of Metazoa
based on the Ctenophora-sister hypothesis are not supported by the currently available data.

### Feuda *et al.* 2017

Feuda *et al.* didn't generate any new data, instead they used the data-recoding methods to reanalyze two key datasets that support Ctenophora-sister (Whelan2015_D20, Chang2015 datasets). It was the first study phylogenomic analysis using recoding strategy to explore animal-root position. The authors compared model adequacy using posterior predictive analyses from a set of site-homogeneous (WAG, LG, GTR, PartitionFinder)and site-heterogeneous (CAT-GTR) models with non-recoding and recoding datasets. The results showed that data-recoding can significant reduce compositional heterogeneity in both datasets with CAT-GTR models and strongly supported Porifera-sister hypothesis. Regarding the recovery of Porifera-sister, the authors concluded:

> Because adequate modeling of the evolutionary process that generated the data is fundamental to recovering an accurate phylogeny, our results strongly support sponges as the sister group of all other animals and provide further evidence that Ctenophora-sister represents a tree reconstruction artifact.

### Whelan and Halalnych 2017a

Whelan *et al.* is the only study to evaluate performance of site-heterogeneous models and site-homogeneous model with data partitioning under the simulation framework. The simulation results suggested that Poisson+CAT model consistently performed worse than other models in simulation datasets. More importantly, the authors also showed that both Poisson + CAT and GTR +  CAT models could overestimated substitutional heterogeneity in many cases. They also reanalyzed datasets from Philippe 2009 and Nosenko 2013 using both CAT models and data partitioning with site -homogeneious model. The results indicated that Poisson + CAT model tends to recover less accurate trees and both GTR + CAT and data partitioning strongly supported Ctenophora-sister in reanalyses. The authors also concluded:

> Practices such as removing constant sites and parsimony uninformative characters, or using CAT-F81 when CAT-GTR is deemed too computationally expensive,
cannot be logically justified. Given clear problems with CAT-F81, phylogenies previously inferred with this model should be reassessed.


### Whelan *et al.* 2017b

Whelan *et al.* added 27 new ctenophore transcriptomic data to explore animal-root position as well as relationships within Ctenophores. It significantly increased ctenophore taxon sampling than other studies. Putative orthologs were determined largely similar to Whelan2015, with the difference of a core Ctenophora core datasets were constructed here with more than 2000 genes. The subsequent filtering strategy was also similar to the previous study. All analyses using site-homogeneous and site-heterogeneous models strongly supported Ctenophora-sister hypothesis, even with CAT-GTR model in Choanoazoa dataset. Regarding the recovery of Ctenophora-sister, the authors concluded:
>Using datasets with reasonably high ctenophore and other non-bilaterian taxon sampling, our results strongly reject the hypothesis that sponges are the sister lineage to all other extant metazoans.

### Simion *et al.* 2017

Simion *et al.* [@Dunn:2008ky] added transcriptomic data for 21 new animals. The data matrix was constructed using a semi-automated approach to comprehensively detect and eliminate potential systematic errors. The resulting dataset comprises 1,719 genes and 97 species, including 61 non-bilaterian metazoan species. It was by far the largest phylogenomic dataset in terms of taxon and gene sampling related to animal-root question.

The supermatrix was first analyzed using the Poisson-CAT model. Different that other phylobayes analyses, Simion *et al.* used a gene jackknife strategy based on 100 analyses to overcome the computational limitation because of the large data size. Each jackknife is baed on a random selection of ~ 25% of the genes. The Phylobayes with site-heterogeneous model strongly supported the Porifera-sister, whereas site-homogeneous strongly supported Ctenophora-sister in the supermatrix dataset. Importantly, the authors compared the behavior of long-branch effect between site-heterogenious and site-homogeneous models by progressively removal taxa and concluded higher sensitivity of site-homogeneous models to LBA than CAT models. Regarding the recovery of Ctenophora-sister, the authors concluded:

> Our dataset outperforms previous metazoan gene superalignments in terms of data quality and quantity. Analyses with a best-fitting site-heterogeneous evolutionary model provide strong statistical support for placing sponges as the sister-group to all other metazoans, with ctenophores emerging as the second-earliest branching animal lineage.




## Model matrix comparison

WAG and LG are both fixed exchange matrices. Their differences are largely limitted to a few amino acid changes.

```{r wagvlg}


# Comparison of LG and WAG matrices

library( tidyverse )
library( magrittr )

# Order in iqtree matrix
aa = c("A", "R", "N", "D", "C", "Q", "E", "G", "H", "I", "L", "K", "M", "F", "P", "S", "T", "W", "Y", "V")

# Functional groups adjacent, from https://en.wikipedia.org/wiki/Amino_acid
aa_functional = c( "R", "H", "K", "D", "E", "S", "T", "N", "Q", "C", "G", "P", "A", "V", "I", "L", "M", "F", "Y", "W" )

triange_to_matrix = function( triangle ){
	R_matrix = rep(NA, 400)
	dim( R_matrix ) = c(20,20)


	rownames( R_matrix ) = aa
	colnames( R_matrix ) = aa

	# Though iqtree specifies lower triangle, actually in the correct order for upper triangle as this R command reads it
	R_matrix [ upper.tri( R_matrix ) ] = triangle

	# Now fill in the lower half of the matrix
	R_matrix[ is.na(R_matrix) ] = t( R_matrix )[ is.na(R_matrix) ]

	# Scale
	R_matrix = R_matrix / mean( R_matrix, na.rm=TRUE )

	# Reorder by functional groupings
	R_matrix = R_matrix[, match(aa_functional, colnames(R_matrix) )]
	R_matrix = R_matrix[match(aa_functional, rownames(R_matrix)), ]

	R_matrix
}

plot_matrix = function( R_matrix, title=NULL ){
	D = data.frame( R_matrix )
	D$source = row.names(D)
	Dg = gather( D, 1:20, key=dest, value=exch, -source )
	Dg %<>%
		mutate( source=factor(source, levels=aa_functional)  ) %>%
		mutate( dest=factor(dest, levels=aa_functional)  )

	Dg %>%
		ggplot(aes(x=dest, y=source)) +
		geom_tile(aes(fill=exch)) +
		scale_fill_gradient2() +
		ggtitle(title)
}

# Triangle matrices from iqtree source code at https://github.com/Cibiv/IQ-TREE/blob/master/model/modelprotein.cpp

wag_triangle = c(55.15710, 50.98480, 63.53460, 73.89980, 14.73040, 542.94200, 102.70400, 52.81910, 26.52560, 3.02949, 90.85980, 303.55000, 154.36400, 61.67830, 9.88179, 158.28500, 43.91570, 94.71980, 617.41600, 2.13520, 546.94700, 141.67200, 58.46650, 112.55600, 86.55840, 30.66740, 33.00520, 56.77170, 31.69540, 213.71500, 395.62900, 93.06760, 24.89720, 429.41100, 57.00250, 24.94100, 19.33350, 18.69790, 55.42360, 3.94370, 17.01350, 11.39170, 12.73950, 3.04501, 13.81900, 39.79150, 49.76710, 13.15280, 8.48047, 38.42870, 86.94890, 15.42630, 6.13037, 49.94620, 317.09700, 90.62650, 535.14200, 301.20100, 47.98550, 7.40339, 389.49000, 258.44300, 37.35580, 89.04320, 32.38320, 25.75550, 89.34960, 68.31620, 19.82210, 10.37540, 39.04820, 154.52600, 31.51240, 17.41000, 40.41410, 425.74600, 485.40200, 93.42760, 21.04940, 10.27110, 9.61621, 4.67304, 39.80200, 9.99208, 8.11339, 4.99310, 67.93710, 105.94700, 211.51700, 8.88360, 119.06300, 143.85500, 67.94890, 19.50810, 42.39840, 10.94040, 93.33720, 68.23550, 24.35700, 69.61980, 9.99288, 41.58440, 55.68960, 17.13290, 16.14440, 337.07900, 122.41900, 397.42300, 107.17600, 140.76600, 102.88700, 70.49390, 134.18200, 74.01690, 31.94400, 34.47390, 96.71300, 49.39050, 54.59310, 161.32800, 212.11100, 55.44130, 203.00600, 37.48660, 51.29840, 85.79280, 82.27650, 22.58330, 47.33070, 145.81600, 32.66220, 138.69800, 151.61200, 17.19030, 79.53840, 437.80200, 11.31330, 116.39200, 7.19167, 12.97670, 71.70700, 21.57370, 15.65570, 33.69830, 26.25690, 21.24830, 66.53090, 13.75050, 51.57060, 152.96400, 13.94050, 52.37420, 11.08640, 24.07350, 38.15330, 108.60000, 32.57110, 54.38330, 22.77100, 19.63030, 10.36040, 387.34400, 42.01700, 39.86180, 13.32640, 42.84370, 645.42800, 21.60460, 78.69930, 29.11480, 248.53900, 200.60100, 25.18490, 19.62460, 15.23350, 100.21400, 30.12810, 58.87310, 18.72470, 11.83580, 782.13000, 180.03400, 30.54340, 205.84500, 64.98920, 31.48870, 23.27390, 138.82300, 36.53690, 31.47300)
wag_matrix = triange_to_matrix( wag_triangle ) %>% log()

lg_triangle = c(0.425093, 0.276818, 0.751878, 0.395144, 0.123954, 5.076149, 2.489084, 0.534551, 0.528768, 0.062556, 0.969894, 2.807908, 1.695752, 0.523386, 0.084808, 1.038545, 0.363970, 0.541712, 5.243870, 0.003499, 4.128591, 2.066040, 0.390192, 1.437645, 0.844926, 0.569265, 0.267959, 0.348847, 0.358858, 2.426601, 4.509238, 0.927114, 0.640543, 4.813505, 0.423881, 0.311484, 0.149830, 0.126991, 0.191503, 0.010690, 0.320627, 0.072854, 0.044265, 0.008705, 0.108882, 0.395337, 0.301848, 0.068427, 0.015076, 0.594007, 0.582457, 0.069673, 0.044261, 0.366317, 4.145067, 0.536518, 6.326067, 2.145078, 0.282959, 0.013266, 3.234294, 1.807177, 0.296636, 0.697264, 0.159069, 0.137500, 1.124035, 0.484133, 0.371004, 0.025548, 0.893680, 1.672569, 0.173735, 0.139538, 0.442472, 4.273607, 6.312358, 0.656604, 0.253701, 0.052722, 0.089525, 0.017416, 1.105251, 0.035855, 0.018811, 0.089586, 0.682139, 1.112727, 2.592692, 0.023918, 1.798853, 1.177651, 0.332533, 0.161787, 0.394456, 0.075382, 0.624294, 0.419409, 0.196961, 0.508851, 0.078281, 0.249060, 0.390322, 0.099849, 0.094464, 4.727182, 0.858151, 4.008358, 1.240275, 2.784478, 1.223828, 0.611973, 1.739990, 0.990012, 0.064105, 0.182287, 0.748683, 0.346960, 0.361819, 1.338132, 2.139501, 0.578987, 2.000679, 0.425860, 1.143480, 1.080136, 0.604545, 0.129836, 0.584262, 1.033739, 0.302936, 1.136863, 2.020366, 0.165001, 0.571468, 6.472279, 0.180717, 0.593607, 0.045376, 0.029890, 0.670128, 0.236199, 0.077852, 0.268491, 0.597054, 0.111660, 0.619632, 0.049906, 0.696175, 2.457121, 0.095131, 0.248862, 0.140825, 0.218959, 0.314440, 0.612025, 0.135107, 1.165532, 0.257336, 0.120037, 0.054679, 5.306834, 0.232523, 0.299648, 0.131932, 0.481306, 7.803902, 0.089613, 0.400547, 0.245841, 3.151815, 2.547870, 0.170887, 0.083688, 0.037967, 1.959291, 0.210332, 0.245034, 0.076701, 0.119013, 10.649107, 1.702745, 0.185202, 1.898718, 0.654683, 0.296501, 0.098369, 2.188158, 0.189510, 0.249313)
lg_matrix = triange_to_matrix( lg_triangle ) %>% log()



lg_wag_diff = lg_matrix - wag_matrix



lg_wag_diff_thresh =lg_wag_diff
lg_wag_diff_thresh[abs(lg_wag_diff_thresh)<1] = 0


grid.arrange(
	plot_matrix(lg_matrix, title="(A) LG matrix"),
	plot_matrix(wag_matrix, title="(B) WAG matrix"),
	plot_matrix(lg_wag_diff, title="(C) LG - WAG"),
	plot_matrix( lg_wag_diff_thresh, title="(D) Thresholded LG - WAG" )
)

```


## Matrix mapping

Taxa and partition correspondence across manuscripts was assessed by comparing all sequences for each taxon in each partition across all matrices with diamond blast. Based on inspection of sequence similarity, we excluded all comparisons with less than 99% identity and greater than 10^-25^ e-value.


### Taxa comparison across matrices

The primary intent of comparing taxa across matrices was to validate our taxon name reconciliation across studies.

We first considered pairwise similarity between the same species from different matrices in different studies.






### Partition comparison across matrices


```{r partition_mapping}

# Investigate connected components of genes (ie partitions) that include multiple members fromt he same matrix
#clusterstats_within_manuscripts = partition_map_global %>% group_by( matrix, component_number ) %>% summarise(n=n()) %>% arrange(desc(n))
#clusterstats = partition_map_global %>% group_by( component_number ) %>% summarise(n=n()) %>% arrange(desc(n))

#clusterstats_within_manuscripts %>% print(n=100)
#clusterstats %>% print(n=100)
#partition_network_summary %>% print()

```

The count for a partition pair can be much larger than the number of genes in the matrix, which suggests that the count is the number of hsps rather than the number of sequences with hits.

There are `r length(unique(partition_network_summary$matrix))` matrices. A gene that is perfectly sampled would form a cluster with this size. To our surprise, very few clusters, though, are this size. This suggests that intersection of genes between matrices is extreme low. This finding suggested genes used for exploring the position of the root of the animal phylogeny





## References
